### PR TITLE
Patch fixed critical CVEs in kserve/alibi-explainer

### PR DESCRIPTION
**Pull request by MessKon**
**What this PR does / why we need it**:
Fixing critical CVEs in latest kserve/alibi-explainer version
Comparing this to latest stable `kserve/alibi-explainer` image:
image

image

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Not sure there's an open issue for this

**Type of changes**
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

**Feature/Issue validation/testing**:

`kserve/alibi-explainer` image builds ok with no issues

- Logs

**Special notes for your reviewer**:

1. This PR only changes image versions

**Release note**:
```release-note
fixing vulnerabilities (CVEs) in kserve/alibi-explainer

```

### CODE REVIEW

File: python/alibiexplainer.Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.7
+FROM python:3.9-slim-bullseye
```
Comment (yuzisun) [inline python/alibiexplainer.Dockerfile]: what's the difference between `3.9-slim-bullseye` and `3.9-slim` ?

Comment (MessKon) [reply python/alibiexplainer.Dockerfile]: whilst not being able to see differences between the builds of the 2 images ([3.9-slim](https://hub.docker.com/layers/python/library/python/3.9-slim/images/sha256-4169ae884e9e7d9bd6d005d82fc8682e7d34b7b962ee7c2ad59c42480657cb1d?context=explore) and [3.9-slim-bullseye](https://hub.docker.com/layers/python/library/python/3.9-slim-bullseye/images/sha256-4169ae884e9e7d9bd6d005d82fc8682e7d34b7b962ee7c2ad59c42480657cb1d?context=explore))

when built with  `python:3.9-slim-bullseye` the image reports 3 less critical CVEs:
image

image

**Comment by MessKon** [comment]
@yuzisun I think this one's close. all tests are passing at the moment

**Comment by yuzisun** [comment]
Thanks @MessKon !
/lgtm
/approve

**Pull request merged**